src/main/java/ca/uhn/fhir/jpa/starter/FhirServerConfigCommon.java

```diff
@@ -77,7 +78,6 @@ public DaoConfig daoConfig(AppProperties appProperties) {
     retVal.setAllowMultipleDelete(appProperties.getAllow_multiple_delete());
     retVal.setAllowExternalReferences(appProperties.getAllow_external_references());
     retVal.setExpungeEnabled(appProperties.getExpunge_enabled());
-    retVal.setAutoCreatePlaceholderReferenceTargets(appProperties.getAllow_placeholder_references());
```

Comment:
Was removing this line deliberate? This setting isn't related to partitioning and is still useful..

Reply:
Yep! Search for `setAutoCreatePlaceholderReferenceTargets` here and you'll see what I'm fixing.

BUT (from what I recall) I couldn't figure out where `getAllow_placeholder_references` was actually meant to go, so for now that setting is dead code.

Reply:
Got it, thanks!